app/src/main/res/values/strings.xml

```diff
@@ -164,6 +164,7 @@
     <string name="youtube_restricted_mode_enabled_title">Turn on YouTube\'s \"Restricted Mode\"</string>
     <string name="youtube_restricted_mode_enabled_summary">YouTube provides a \"Restricted Mode\" which hides potentially mature content</string>
     <string name="restricted_video">This video is age restricted.\n\nTurn on \"%1$s\" in the settings if you want to see it.</string>
+    <string name="restricted_video_no_stream">This video is age restricted and no stream could be fetched.</string>
```

Comment:
Wouldn't it be better to say something like `Due to new YouTube policies with age-restricted videos, this video cannot be streamed in NewPipe`?

Reply:
Well there are still age-restricted videos which can be played. That's why I wanted to give a reason why this video in particular cannot be played in difference to the other age-restricted videos. Maybe we can combine both error messages:
`This video is age-restricted\nDue to new YouTube policies with age-restricted videos, NewPipe cannot access any of its video streams and thus is unable to play it.`

Reply:
I will push a commit